### ISSUE TITLE
`resource/pingdirectory_default_web_application_extension`: Error: Provider produced inconsistent result after apply

### ISSUE THREAD

**Issue by patrickcping** [pingidentity/terraform-provider-pingdirectory#305]
### PingDirectory Terraform provider Version
Check the version you have configured in your .tf files. If you are not running the latest version of the provider, please upgrade because your issue may have already been fixed.

`v1.0.0`

### PingDirectory Version
Check the version of the PingDirectory server being configured.

`9.3.0.0`

### Terraform Version
Run `terraform -v` to show the version.

```
Terraform v1.5.7
on darwin_amd64
```

### Affected Resource(s)
Please list the resources as a list, for example:
- `resource/pingdirectory_default_web_application_extension`

### Terraform Configuration Files
```hcl
# Copy-paste your Terraform configurations here - for large Terraform configs,
# please use a service like Dropbox and share a link to the ZIP file. For
# security, you can also encrypt the files using our GPG public key.

# Remember to replace any account/customer sensitive information in the configuration before submitting the issue

resource "pingdirectory_default_web_application_extension" "console_web_application_extension" {
  name = "Console"
  description = "The Administrative Console"
  base_context_path = "/console"
  war_file = "webapps/console.war"

  ldap_server = "ldaps://0d7334259c96:1636"
  log_file = "logs/webapps/console.log"
  complexity = "ADVANCED"

  sso_enabled        = true
  oidc_client_id     = pingone_application.pingdirectory_admin_console.oidc_options[0].client_id
  oidc_client_secret = pingone_application.pingdirectory_admin_console.oidc_options[0].client_secret
  oidc_issuer_url    = module.pingone_utils.pingone_environment_issuer
}

```

### Debug Output
Please provide your debug output with `TF_LOG=DEBUG` enabled on your `terraform plan` or `terraform apply`. Please provide a link to a GitHub Gist containing the complete debug output: https://www.terraform.io/docs/internals/debugging.html. Please do NOT paste the debug output in the issue; just paste a link to the Gist.

```
Error: Provider produced inconsistent result after apply
│ 
│ When applying changes to pingdirectory_default_web_application_extension.console_web_application_extension, provider
│ "provider[\"registry.terraform.io/pingidentity/pingdirectory\"]" produced an unexpected new value: .complexity: was
│ cty.StringVal("ADVANCED"), but now cty.StringVal("advanced").
│ 
│ This is a bug in the provider, which should be reported in the provider's own issue tracker.
```

### Panic Output
If Terraform produced a panic, please provide your debug output from the GO panic

N/a

### Expected Behavior
What should have happened?

Either:
- ModifyPlan to convert the case insensitivity on the provider schema to the case sensitive value expected by the service
- Validation on the schema to enforce proper case sensitivity (this may actually be an enum field)

### Actual Behavior
What actually happened?

Inconsistent plan error as shown.

Documentation may need an update to show the valid values for the field if an enum, and possibly an example HCL block for the `Console` default application

### Steps to Reproduce
Please list the steps required to reproduce the issue, for example:
1. `terraform apply`

### References
Are there any other GitHub issues (open or closed) or Pull Requests that should be linked here? For example:

**Comment by henryrecker-pingidentity** [comment]
Looks like PD is auto-formatting this attribute. For a workaround for now you should be able to use lower case "advanced".

**Comment by henryrecker-pingidentity** [comment]
Resolved with #337 , will be included in the next minor release
